src/components/App.js

```diff
@@ -33,6 +41,13 @@ function App() {
           setTodaysSeconds={setTodaysSeconds}
           todaysMinutes={todaysMinutes}
           setTodaysMinutes={setTodaysMinutes}
+          weeklyAverageShowerTime={weeklyAverageShowerTime}
```

Comment:
Props for days. I feel like this app could be a possible use case for global state management, but that seems like a lot to try to integrate after this much functionality.